packages/components/src/components/domainproperties/DomainRow.tsx

```diff
@@ -385,58 +375,58 @@ export class DomainRow extends React.PureComponent<IDomainRowProps, IDomainRowSt
                         name={createFormInputName(DOMAIN_FIELD_ADV)}
-                        id={createFormInputId(DOMAIN_FIELD_ADV, index)}
+                        id={createFormInputId(DOMAIN_FIELD_ADV, domainIndex, index)}
                         onClick={this.onShowAdvanced}
                         className="domain-row-button"
                     >
                         Advanced Settings
-                    </Button>}
-                </>
+                    </Button>
                 )}
-                <div className="field-icon" id={createFormInputId(DOMAIN_FIELD_EXPAND, index)} onClick={this.onExpand}>
-                    <FontAwesomeIcon size='lg' color={(dragging || hover) ? HIGHLIGHT_BLUE : NOT_HIGHLIGHT_GRAY}
-                                     icon={expanded ? faMinusSquare : faPlusSquare}/>
-                </div>
+                {!(isFieldFullyLocked(field.lockType) || isFieldPartiallyLocked(field.lockType)) &&
+                    <DeleteIcon
+                        id={createFormInputId(DOMAIN_FIELD_DELETE, domainIndex, index)}
+                        title={'Remove field'}
+                        iconCls={'domain-field-delete-icon'}
+                        onDelete={this.onDelete}
+                    />
+                }
+                <FieldExpansionToggle
+                    cls={'domain-field-expand-icon'}
+                    expanded={expanded}
+                    expandedTitle={"Hide additional field properties"}
+                    collapsedTitle={"Show additional field properties"}
+                    id={createFormInputId(DOMAIN_FIELD_EXPAND, domainIndex, index)}
+                    onClick={this.onExpand}
+                />
             </div>
         )
     }
 
     render() {
         const { closing, isDragDisabled, showAdv, showingModal } = this.state;
-        const { index, field, expanded, expandTransition, fieldError, maxPhiLevel, dragging, domainId,
+        const { index, field, expanded, expandTransition, fieldError, maxPhiLevel, dragging, domainId, domainIndex,
             helpNoun, showDefaultValueSettings, defaultDefaultValueType, defaultValueOptions, appPropertiesOnly } = this.props;
 
         return (
-            <Draggable draggableId={createFormInputId("domaindrag", index)} index={index} isDragDisabled={showingModal || isDragDisabled}>
+            <Draggable draggableId={createFormInputId("domaindrag", domainIndex, index)} index={index} isDragDisabled={showingModal || isDragDisabled}>
                 {(provided) => (
                     <div className={this.getRowCssClasses(expanded, closing, dragging, fieldError)}
                          {...provided.draggableProps}
                          ref={provided.innerRef}
                          tabIndex={index}
-                         onMouseEnter={showingModal ? undefined : this.onMouseOver}
-                         onMouseLeave={showingModal ? undefined : this.onMouseOut}
```

Comment:
Be careful with this, there's some funky interplay with modals and the mouse over stuff, causing modal re-render.

Reply:
@labkey-martyp I tried testing this and am not seeing any issues. Do you recall the repro steps? Do I try to drag a row when a modal is open, or try to click something while or modal is open, or something else?

Reply:
Oh looks like we're just removing the mouse-over highlighting all together.  In that case its probably not an issue.  The issue was if mousing over the background fields when a modal was open the modal would re-render when the mouseover events fired.  Was messing with tests and looked weird

Reply:
Ah ok, yeah that would make sense. That shouldn't be an issue anymore, as you said, with the removal of the mouseover handlers.